src/components/Sort/__mocks__/mockSortExample.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import { Sort, Toolbar } from '../../../index';
+
+const bindMethods = (context, methods) => {
+  methods.forEach(method => {
+    context[method] = context[method].bind(context);
+  });
+};
+
+export const mockSortFields = [
+  {
+    id: 'name',
+    title: 'Name',
+    isNumeric: false
+  },
+  {
+    id: 'address',
+    title: 'Address',
+    isNumeric: false
+  },
+  {
+    id: 'birthMonth',
+    title: 'Birth Month',
+    isNumeric: true
+  },
+  {
+    id: 'car',
+    title: 'Car',
+    isNumeric: false
+  }
+];
+
+export class MockSortExample extends React.Component {
+  constructor() {
+    super();
+
+    bindMethods(this, ['updateCurrentSortType', 'toggleCurrentSortDirection']);
+
+    this.state = {
+      currentSortType: mockSortFields[0],
+      isSortNumeric: mockSortFields[0].isNumeric,
+      isSortAscending: true
+    };
+  }
+
+  filterAdded = (field, value) => {
+    let filterText = '';
+    if (field.title) {
+      filterText = field.title;
+    } else {
+      filterText = field;
+    }
+    filterText += ': ';
+
+    if (value.filterCategory) {
+      filterText +=
+        (value.filterCategory.title || value.filterCategory) +
+        '-' +
+        (value.filterValue.title || value.filterValue);
+    } else if (value.title) {
+      filterText += value.title;
+    } else {
+      filterText += value;
+    }
+
+    let activeFilters = [...this.state.activeFilters, { label: filterText }];
+    this.setState({ activeFilters: activeFilters });
+  };
+
+  updateCurrentSortType(sortType) {
+    const { currentSortType } = this.state;
+    if (currentSortType !== sortType) {
+      this.setState({
+        currentSortType: sortType,
+        isSortNumeric: sortType.isNumeric,
+        isSortAscending: true
+      });
+    }
+  }
+
+  toggleCurrentSortDirection() {
+    const { isSortAscending } = this.state;
+
+    this.setState({ isSortAscending: !isSortAscending });
```

Comment:
simple toggles like this can be simplified to:
```
    this.setState(prevState => {
      isSortAscending: !prevState.isSortAscending;
    });
```
as setState also accepts a function ;)
https://medium.com/@shopsifter/using-a-function-in-setstate-instead-of-an-object-1f5cfd6e55d1